server/apps/distributed-app/sample-configuration/rabbitmq.properties

```
@@ -73,3 +73,8 @@ mailqueue.size.metricsEnabled=false
 # Disable with caution (this only makes sense in a distributed setup where other nodes consume tasks).
 # Defaults to true.
 task.consumption.enabled=true
+
+# Configure queue ttl (in ms). References: https://www.rabbitmq.com/ttl.html#queue-ttl.
+# This is used only on queues used to share notification patterns, are exclusive to a node. If omitted, it will not add the TTL configure when declaring queues.
+# Optional integer, defaults is 3600000.
+notification.queue.ttl=3600000
```

```suggestion
#notification.queue.ttl=3600000
```

Let's not activate this by default